seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/writer/TextWriteStrategy.java

```diff
@@ -76,6 +82,9 @@ public void setSeaTunnelRowTypeInfo(SeaTunnelRowType seaTunnelRowType) {
                         .dateFormatter(dateFormat)
                         .dateTimeFormatter(dateTimeFormat)
                         .timeFormatter(timeFormat)
+                        .enableHiveCollectionType(enableHiveCollectionType)
```

Comment:
I think you can create a `HiveTextSerializationSchema` here when the sink is `Hive`.